docs/performance.md

```diff
@@ -1,7 +1,543 @@
+available via an export of a centrally managed shared file system using [NFS](https://en.wikipedia.org/wiki/Network_File_System).
+In this particular case we evaluated TensorFlow start-up time on a worker node of the HPC-UGent Tier-2 cluster
+*victini*, which uses [NFS-Ganesha](https://nfs-ganesha.github.io).
+
+<p align="center">
+<img src="../img/perf/tensorflow-startup-nfs.png" alt="Start-up performance of TensorFlow: NFS" width="70%"/></br>
+</p>
+
+Here we already observe a significant, yet still somewhat limited, increase in start-up time.
+
+With a [cold cache](#cold_cache), ~6.4 seconds are required to start TensorFlow, and there is notably
+larger variation in the timings; neither of these observations is a surprise with NFS.
+
+The start-up time with a [hot cache](#hot_cache) is also significantly higher compared to using a local
+filesystems at ~3.8 seconds, but since this is an older
+generation system we can not directly compared these timings to the ones observed for local disk and RAM disk
+earlier. There is a significant penalty involved though when using NFS, since on the same system we observed start-up times
+of ~2.5 seconds when using RAM disk, as well as with a hot kernel file system cache.
+
+While these timings are still quite OK, it worth noting that NFS is notorious for not scaling well when it needs
+to serve many different client systems, which is not apparent in these results since only a single client system
+was used.
+
+#### Lustre
+
+[Lustre](https://www.lustre.org) is a popular open-source distributed parallel file system that is used on many large-scale HPC clusters,
+including [VSC Tier-1 Hortense](https://docs.vscentrum.be/gent/tier1_hortense.html) and flagship EuroHPC pre-exascale system [LUMI](https://docs.lumi-supercomputer.eu).
+
+It is often also the *only* shared file system that is available on these systems,
+and hence is the most obvious choice to make software installations available to the cluster worker nodes.
+
+<p align="center">
+<img src="../img/perf/tensorflow-startup-lustre.png" alt="Start-up performance of TensorFlow: Lustre" width="100%"/></br>
+</p>
+
+We considered two different ways of accessing the Lustre file system of VSC Tier-1 Hortense
+([see below](#system_configurations) for more setails) on which TensorFlow was installed:
+
+* directly, via a read-write mount point;
+* indirectly, via a read-only mount point ([which is recommended on Hortense](https://docs.vscentrum.be/gent/tier1_hortense.html#accessing-software-via-readonly-mount-point));
+
+In both cases, the start-up timings were quite similar, with a dramatic increase to ~75 seconds
+in the [cold cache](#cold_cache) situation, and a very reasonable ~2.8-2.9 seconds with a [hot cache](#hot_cache).
+
+The startup-time of well over 1 minute for TensorFlow with a cold cache is a clear sign that (this particular version and configuration of)
```

Comment:
"of well over 1 minute" -- for me this looks like spoken english, not written english. 

maybe something like:
TensorFlow's cold cache performance with having a significant start-up time of >1min is a clear sign...